### PR TITLE
Add missing support for DSHOT_BBANG arming disabled flag

### PR DESCRIPTION
**Pull request by etracer65**
Part of BF4.1 so should be added to 10.6.

### CODE REVIEW

File: src/js/tabs/setup.js [hunk 1]
```diff
@@ -237,6 +237,7 @@ TABS.setup.initialize = function (callback) {
             if (semver.gte(CONFIG.apiVersion, "1.42.0")) {
                 disarmFlagElements.splice(disarmFlagElements.indexOf('THROTTLE'), 0, 'CRASH');
                 disarmFlagElements = disarmFlagElements.concat(['REBOOT_REQD']);
+                disarmFlagElements = disarmFlagElements.concat(['DSHOT_BITBANG']);
```
Comment (McGiverGim) [inline src/js/tabs/setup.js]: I think is better to join the concat to the earlier element:
```
disarmFlagElements = disarmFlagElements.concat(['REBOOT_REQD'],
                                               ['DSHOT_BITBANG']);
```
I know we have code in both ways but I think is cleaner like this.

Comment (etracer65) [reply src/js/tabs/setup.js]: Ok.

Comment (McGiverGim) [reply src/js/tabs/setup.js]: Both are the same 😋 or at least I don't see difference.

Comment (etracer65) [reply src/js/tabs/setup.js]: Well the difference is:
```
...disarmFlagElements.concat(['REBOOT_REQD'],['DSHOT_BITBANG']);
```
vs.
```
...disarmFlagElements.concat(['REBOOT_REQD', 'DSHOT_BITBANG']);
```
Maybe doesn't matter and I should probably just be consistent with the other existing code.

**Comment by Docteh** [comment]
I get good use of firmware/configurator cross references, this is related to https://github.com/betaflight/betaflight/pull/8779 

Today's https://github.com/betaflight/betaflight/pull/9022 doesn't require coordination

File: locales/en/messages.json [hunk 2]
```diff
@@ -918,6 +918,10 @@
         "message": "A configuration change requires a reboot",
         "description": "Message that pops up to describe the REBOOT_REQD arming disable flag"
     },
+    "initialSetupArmingDisableFlagsTooltipDSHOT_BBANG": {
+        "message": "Bitbang DSHOT is not working properly and the motors can't be controlled. Likely caused by a timer conflict with other features enabled on the flight controller.",
```
Comment (mikeller) [inline locales/en/messages.json]: Should probably be 'bitbanged Dshot'.

Comment (etracer65) [reply locales/en/messages.json]: Updated.

**Pull request merged**